testing/python/metafunc.py

```diff
@@ -1454,9 +1454,9 @@ def test_hello(arg):
             def test_world(arg):
                 assert arg in (1,2)
             def test_checklength():
-                assert len(values) == %d
+                assert len(values) == {length}
         """
-            % (scope, length)
+            % scope
```

Comment:
Could we also use `scope` inside the fstring?

Reply:
``scope:r`` gives an error when it's a string (string has no ``__repr__``) and a test is failing without it. (I had to bisect all the line one bye one to realize this 😄 ). Maybe ``scope!r``, I'll try that.

Reply:
I don't know why I thought it was ``:r`` now that I searched the difference I didn't find ``:r`` anywhere.